dashboard/test/lib/services/user/password_resetter_test.rb

```diff
@@ -8,18 +8,32 @@ class Services::User::PasswordResetterTest < ActiveSupport::TestCase
   describe '#call' do
     subject(:reset_password) {described_instance.call}
 
-    let!(:user) {create(:user, email: email)}
     let(:mail) {ActionMailer::Base.deliveries.first}
 
-    it 'returns user' do
```

Comment:
It looks like the simple case was removed. Was this intentional? Which test is for the simple, happy case `let!(:user) {create(:user, email: email)}`? Or maybe this test case is redundant given the below test cases?

Reply:
I initially felt it was redundant, but it wouldn't hurt to have. I'll add it back in.